IntersonArrayDeviceRF.hxx

```diff
@@ -141,11 +144,42 @@ public:
       {
       Stop();
       success = hwControls.SendHighVoltage( voltage, voltage );
+      highVoltage = voltage;
       Start();
       }
     return success;
     }
 
+  bool SetFrequencyAndVoltage( unsigned char fIndex, unsigned char voltage )
+    {
+    if( fIndex == frequencyIndex && voltage == highVoltage )
```

Comment:
In my experience changing the frequency was much slower than changing the voltage. I'd suggest to check for them independently and only do the calls that are necessary on the probe.

Reply:
I added this function that optionally changes both Frequency and Voltage within a single start/stop pair.   My thought was that if Frequency and Voltage both needed to be changed, then it was better to do Stop, SetVolt, SetFreq, Start rather than Stop, SetVol, Start, Stop, SetFreq, Start.   There seems to be a time cost associated with starting and stopping the probe, and we often need to change both Freq and Power at the same time, so I picked these two settings as the ones to combine to provide a speedup.  Admittedly the speedup is anecdotal, but the cost to change Freq + Power in one start/stop pair seems to be close to the cost of changing just Freq.